environs/config/config.go

```diff
@@ -990,6 +1011,20 @@ func (c *Config) CAPrivateKey() (key string, ok bool) {
 	return "", false
 }
 
+// RsyslogConfig return the rsyslog client config.
+func (c *Config) RsyslogConfig() (*rsyslog.ClientConfig, error) {
```

Comment:
not quite sure i agree on this point.. whoever is interested in the value should check the returned error.